packages/nimble/inst/include/nimble/nimbleEigenNimArr.h

```diff
@@ -86,7 +86,7 @@ void setValuesIndexRange(NimArrBase<T> &nimArr, ManyVariablesMapAccessor &MVA, c
 
 
 template<typename Derived>
-double calculate(NodeVectorClassNew &nodes, const Derived &indices, bool logical=false) {
+double calculate(NodeVectorClassNew &nodes, const Derived &indices, bool logical) { // remove default value of logical to fix function overloading issues
```

Comment:
Specifically, the function does not "remove anything", it is your PR that "removes" something.